### ISSUE TITLE
Train a new Alog on the new 1.6 million Tweets

### ISSUE THREAD

**Issue by parkerburchett** [parkerburchett/NaturalLanguage#25]
See the kaggle dataset in "Datasets"

**Comment by parkerburchett** [comment]
You probably want to use a linear Support Vector Machine

**Comment by parkerburchett** [comment]
I ended up using the sklearn stochastic gradient descent method. The highest time cost was converting the tweets to boolean vectors.  After that training the SGDClassifier() was very fast. approx 2 minutes when 10,000 features.
